### ISSUE TITLE
TYPO3 V11: Export Button -> No effect/function

### ISSUE THREAD

**Issue by micha68g** [pagemachine/typo3-formlog#122]
Hi,
I don't know if I maybe missed some configuration work, but the "export"-Button has simply no function when clicked. (TYPO3 11.5.5., formlog 2.0.0).
Formlog entries get listed correctly, even with my extra-configuration for the list-view...but I can't export anything (even in basic mode before any changes to the list-settings). Clicking the export-button has just no effect.... (Tested in Chrome, FF and Safari, no js-errors in the console)

```
tx_formlog {
    settings { 
     list {
#some list config -> correctly displayed in "Form log entries" table-view
     }
  }
}
```

Thanks for any hints.
Michael

**Comment by mbrodala** [comment]
Thanks a lot for the report. It seems like a whole bunch of change have gone MIA. (See e.g. the broken pagination)

I'll try to redo these.

**Comment by mbrodala** [comment]
@micha68g I tried my best to catch all lost updates with https://github.com/pagemachine/typo3-formlog/pull/123, can you test this?

**Comment by micha68g** [comment]
Thanks, BE-view is ok now. Broken pagination fixed and export-Button toogles out the csv- and excel-options.
But.... I get 503 errors "Class 'League\Csv\Writer' not found" respectively "Class 'OneSheet\Style\Style' not found" when trying to start an export...

Note:  whole project and extensions were installed *not* using composer....

**Comment by mbrodala** [comment]
Yes, the bundling of Composer libraries happens right before TER upload on release. So this error cannot be avoided on Legacy Mode and a Git clone.

So thanks for testing!

**Comment by mbrodala** [comment]
This was released with version [2.0.1](https://github.com/pagemachine/typo3-formlog/releases/tag/2.0.1) now.
